packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/EditContract.jsx

```diff
@@ -178,6 +185,21 @@ const EditContract = props => {
               disabled
               variant={fieldVariant}
             />
+            {policy.setSync && flag('harvest.toggle-contract-sync') ? (
+              <FieldContainer variant={fieldVariant}>
+                <Label for="account-sync-switch">
+                  {t('contractForm.imported')}
+                </Label>
+                <span>
```

Comment:
can't we use a fragment here?

Reply:
No, we need a span otherwise the contract switch behaves weirdly. I think the Switch must not be a flex child.